### PR TITLE
feature/1611 - Switch select report dialog from 'contributor' to 'election'

### PR DESCRIPTION
**Pull request by sasha-dresden**
Issue [FECFILE-1611](https://fecgov.atlassian.net/browse/FECFILE-1611)

### CODE REVIEW

**Review by toddlees — CHANGES_REQUESTED** [review]
This was missed in the writing of the ticket and I will address it.  For reattributions, we need to say contributor and for redesignations, we need to say election

**Comment by sasha-dresden** [comment]
Updated code so it will use the appropriate text based off whether it's reattribution or redesignation.

File: front-end/src/app/reports/transactions/transaction-list/select-report-dialog/select-report-dialog.component.ts [hunk 1]
```diff
@@ -56,4 +56,8 @@ export class SelectReportDialogComponent implements OnInit {
     this.transaction = undefined;
     this.selectReportDialog?.nativeElement.close();
   }
+
+  get electContrib(): string {
```
Comment (toddlees) [inline front-end/src/app/reports/transactions/transaction-list/select-report-dialog/select-report-dialog.component.ts]: It's a little tricky following the use and what's defined with these getters.  What do you think of these naming changes:
```
  // Label for action the user is trying to perform
  get actionLabel(): string {
    return ReattRedesUtils.isReattribute(this.type) ? 'reattribute' : 'redesignate';
  }

  // URL parameter to set the action type
  get urlParameter(): string {
    return ReattRedesUtils.isReattribute(this.type) ? 'reattribution' : 'redesignation';
  }

  // Label for the target of the action
  get actionTargetLabel(): string {
    return ReattRedesUtils.isReattribute(this.type) ? 'contributor' : 'election';
  }
```

**Review by toddlees — CHANGES_REQUESTED** [review]

File: front-end/src/app/reports/transactions/transaction-list/select-report-dialog/select-report-dialog.component.ts (hunk 1 above)
Comment (sasha-dresden) [reply front-end/src/app/reports/transactions/transaction-list/select-report-dialog/select-report-dialog.component.ts]: Good idea. Updated them with that. Also updated the spec to cover these a bit more fully.

**Review by toddlees — APPROVED** [review]

**Pull request merged**